src/ehrdata/dt/dataloader.py

```diff
@@ -16,11 +16,11 @@
 
 COMPRESSION_FORMATS = Literal["tar.gz", "gztar", "zip", "tar", "gz", "bz", "xz"]
 COMPRESSION_FORMATS_LIST = list(get_args(COMPRESSION_FORMATS))
-RAW_FORMATS = Literal["csv", "txt", "parquet", "h5ad"]
+RAW_FORMATS = Literal["csv", "txt", "parquet", "h5ad", "zarr"]
 RAW_FORMATS_LIST = list(get_args(RAW_FORMATS))
 
 
-def download(
+def _download(
```

Comment:
I remember my suggestion but it's probably cleaner if the whole dataloader module is private? Users should never interact with it, right?